spec/grape/exceptions/base_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe Grape::Exceptions::Base do
+  describe '#compose_message' do
+    subject { described_class.new.send(:compose_message, key, attributes) }
+
+    let(:key) { :invalid_formatter }
+    let(:attributes) { { klass: String, to_format: 'xml' } }
+
+    context 'when I18n enforces available locales' do
+      before { I18n.enforce_available_locales = true }
+      after do
+        I18n.available_locales = %i[de en]
+        I18n.enforce_available_locales = false
+      end
```

Comment:
These probably mess up global state, so the entire spec here should have some kind of after block that restores I18n options to default rather than to `de/en`.

Reply:
@dblock How about the enforce_available_locales setting? As far as I can see, the default setting for this is `true`, so should that be the default setting for all specs too?

Reply:
Sounds like it. If you want to be sure set it globally in spec_helper.rb.